docs/source/probnum.mplstyle

```diff
@@ -74,7 +74,7 @@ legend.fancybox      : False     ## if True, use a rounded box for the
 #### FIGURE
 ## See http://matplotlib.org/api/figure_api.html#matplotlib.figure.Figure
 figure.figsize       : 8, 4.5     ## figure size in inches
-figure.dpi           : 150        ## figure dots per inch
+figure.dpi           : 100        ## figure dots per inch
```

Comment:
there is no real point in a value > 100 here, since jupyter kind of ignores it anyway.